services/app-web/src/pages/StateSubmission/RateDetails/PackagesWithSharedRates/PackagesWithSharedRates.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react'
+
+import { useIndexHealthPlanPackagesQuery } from '../../../../gen/gqlClient'
+import { recordJSException } from '../../../../otelHelpers'
+import { dayjs } from '../../../../common-code/dateHelpers'
+import { getCurrentRevisionFromHealthPlanPackage } from '../../../../gqlHelpers'
+import type { PackageOptionType } from '../../../../components/Select'
+import { useStatePrograms } from '../../../../hooks'
+import { packageName } from '../../../../common-code/healthPlanFormDataType'
+import { FieldYesNo, PoliteErrorMessage } from '../../../../components'
+import { FormGroup, Label, Link } from '@trussworks/react-uswds'
+import { PackageSelect } from '../../../../components/Select'
+import { getIn, useFormikContext } from 'formik'
+import {
+    RateCertFormType,
+    RateInfoArrayType,
+} from '../SingleRateCert/SingleRateCert'
+
+import styles from '../../StateSubmissionForm.module.scss'
+
+export type PackagesWithSharedRatesProps = {
+    index: number
+    key: string
+    shouldValidate: boolean
+    fieldNamePrefix: string
+    parentSubmissionID: string
+}
+
+export const PackagesWithSharedRates = ({
+    index,
+    key,
```

Comment:
`PackagesWithSharedRates` key prop needs to be renamed maybe to `keyProp`.

```suggestion
    keyProp,
```

![Screenshot 2023-06-26 at 10 37 17 PM](https://github.com/Enterprise-CMCS/managed-care-review/assets/98117700/16b986d5-0e64-4d6b-af9a-f0562e0aad49)

Reply:
Fixing this prop might fix the regression in displaying selected shared rates.